src/test/java/org/sagebionetworks/bridge/config/PropertiesConfigTest.java

```diff
@@ -87,6 +88,12 @@ public void testGetInt() throws IOException {
     }
 
     @Test
+    public void testGetIntFromStringPath() throws IOException {
+        Config config = new PropertiesConfig(TEST_CONF_FILE, configFile);
+        assertEquals(2000, config.getInt("example.timeout"));
+    }
```

Comment:
Can we also add test cases for an env-specific config (see testGetEnvSpecific()), as well as a test to verify that the user config overwrites the default config?